pkg/costattribution/manager.go

```diff
@@ -14,28 +14,40 @@ import (
 	"github.com/go-kit/log"
 	"github.com/grafana/dskit/services"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/grafana/mimir/pkg/costattribution/costattributionmodel"
 	"github.com/grafana/mimir/pkg/util/validation"
 )
 
 const (
-	trackerLabel       = "tracker"
-	tenantLabel        = "tenant"
-	defaultTrackerName = "cost-attribution"
-	missingValue       = "__missing__"
-	overflowValue      = "__overflow__"
+	trackerLabel        = "tracker"
+	tenantLabel         = "tenant"
+	reasonLabel         = "reason"
+	defaultTrackerName  = "cost-attribution"
+	missingValue        = "__missing__"
+	overflowValue       = "__overflow__"
+	activeSeriesTracker = "active-series"
+	samplesTracker      = "samples"
 )
 
+type cachedManagerMetricsMetadata struct {
+	sampleTrackerCardinalityDesc       *metricMetadata
+	sampleTrackerOverflowDesc          *metricMetadata
+	activeSeriesTrackerCardinalityDesc *metricMetadata
+	activeSeriesTrackerOverflowDesc    *metricMetadata
+}
```

Comment:
Why are we touching any of these in this PR? These can't fail, right? They don't have user-input label names.